x/claim/module.go

```diff
@@ -170,6 +174,31 @@ func (am AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {}
 
 // EndBlock executes all ABCI EndBlock logic respective to the claim module. It
 // returns no validator updates.
-func (am AppModule) EndBlock(_ sdk.Context, _ abci.RequestEndBlock) []abci.ValidatorUpdate {
+func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.ValidatorUpdate {
+
+	airdropSupply, found := am.keeper.GetAirdropSupply(ctx)
+	if !found || !airdropSupply.IsPositive() {
+		return []abci.ValidatorUpdate{}
+	}
+
+	decayInfo := am.keeper.DecayInformation(ctx)
+	if decayInfo.Enabled && ctx.BlockTime().After(decayInfo.DecayEnd) {
+		err := am.distrKeeper.FundCommunityPool(
+			ctx,
+			sdk.NewCoins(airdropSupply),
+			am.accountKeeper.GetModuleAddress(types.ModuleName))
+		if err != nil {
+			ctx.Logger().Error(
+				fmt.Sprintf("error sending remaining airdrop supply to community pool: %s",
+					err.Error()),
+			)
+		}
+	}
```

Comment:
We should wrap this logic under a method like `EndAirdrop` in `keeper` so we can add tests for the specific logic